javalib/src/main/scala/java/util/concurrent/ConcurrentHashMap.scala

```diff
@@ -62,7 +62,10 @@ class ConcurrentHashMap[K, V] private (initialCapacity: Int, loadFactor: Float)
     inner.clear()
 
   override def keySet(): ConcurrentHashMap.KeySetView[K, V] =
-    new ConcurrentHashMap.KeySetView[K, V](inner.keySet())
+    keySet(null.asInstanceOf[V])
```

Comment:
Actually, this usage seems to be not allowed: Doc says that `keySet(V)` throws NPE if `mappedValue` is `null`. We should probably honor that specification to avoid an accidental `null` to be interpreted as a sentinel.